lib/helpers/isBsonType.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
-const get = require('./get');
-
 /*!
  * Get the bson type, if it exists
  */
 
 function isBsonType(obj, typename) {
-  return get(obj, '_bsontype', void 0) === typename;
+  return (
+    typeof obj === 'object' &&
+    obj !== null &&
+    obj._bsontype === typename
```

Comment:
During my previous PR I was wondering whether or not to add an additional check, eg ` && obj.constructor.name !== 'Object'` or ` && obj.constructor.name.localeCompare(typename, undefined, { sensitivity: 'accent' })`, if someone ever decides to store json data with `{_bsonType: xxx}` inside

Reply:
I dont think that this is necessary. The code change here is functionally equivalent to the previous code. If somebody really starts to create objects with their own _bsonType attribute, than it is their issue. ;)